calendar.md

```diff
@@ -3,9 +3,17 @@ layout: page
 title: Kalendarz
 description: Kalendarium wydarzeń.
 permalink: /calendar/
-#menus: header
+menus: header
 ---
 
 Jeżeli chcesz zorganizować jakieś wydarzenie, zapoznaj się z [poradnikiem na wiki](https://wiki.hsp.sh/zrob_se_meetup)
 
-<iframe src="https://wydarzenia.hsp.sh/" style="border: 0;  max-width: 100%; width:800px; height: 1200px;background: white" frameborder="0" scrolling="no"></iframe>
\ No newline at end of file
+<iframe src="https://gra0007.github.io/modern-cal-embed/iframe.html?ical=https%3A%2F%2Fwydarzenia.hsp.sh%2F%40hs%2Ffeed%2Fics&title=1&nav=1&date=1&view=1&details=0&monstart=1&dview=0&color=%2300cc99&colorbg=%231b1b1b&colortxt=%23ffffff&colorsecondarytxt=%23FFFFFF"
+			width="100%" height="400px" style="border: 1px solid #ffffff"></iframe>
+
+
+Link do kalendarza w formacie: [ICS/WebCal](/calendar.ics), [RSS/Atom](/calendar.atom). Możesz dodać je do swojego kalendarza lub klienta RSS!
+
+`https://hsp.sh/calendar.ics`
+
+`https://hsp.sh/calendar.atom`
```

Comment:
unnecessary

Reply:
Not everyone might realize that you can right click copy the URL. Mind you, that the URLs don't open in new tab, rather download as a file.

Reply:
Lets instruct user to copy the URL and teach them something new about the web. We could link them to a guide (external, or wiki?).

Reply:
That's more complicated than simply putting the URL in plain text to view. The wiki instruction can always be added later.

Reply:
Then maybe remove `<a>` links and place two read only `<input type="text" readonly>` with URLs as values. That would communicate purpose clearly and simplify selecting and copying.